jujugui/static/gui/src/app/components/generic-input/generic-input.js

```diff
@@ -25,10 +25,13 @@ YUI.add('generic-input', function() {
     propTypes: {
       autocomplete: React.PropTypes.bool,
       disabled: React.PropTypes.bool,
+      errors: React.PropTypes.bool,
```

Comment:
s/errors/errorsDisplayed/ or similar so that it seems a boolean and not a sequence?